shared/profile/post-proof-container.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+import React, {Component} from 'react'
+import PostProof from './post-proof'
+import {bindActionCreators} from 'redux'
+import {connect} from 'react-redux'
+import {cancelAddProof, checkProof, outputInstructionsActionLink} from '../actions/profile'
+
+class PostProofContainer extends Component<void, any, void> {
+  static parseRoute (currentPath, uri) {
+    return {
+      componentAtTop: {
+        title: '',
+      },
+    }
+  }
+
+  render () {
+    return <PostProof {...this.props} />
+  }
+}
+
+export default connect(
+  state => {
+    const profile = state.profile
+    return {
+      isOnCompleteWaiting: profile.waiting,
+      platform: profile.platform,
+      errorMessage: profile.error,
+      platformUserName: profile.username,
+      proofText: profile.proof,
+      onCancelText: 'Cancel',
+    }
+  },
+  dispatch => (
+    bindActionCreators({
```

Comment:
if you do use the typed connector it's important to note you can't use bindActionCreators because it's actually pretty hard to type (not sure if flow can even do it right), so just a heads up